Formula/httpie.rb

```diff
@@ -3,38 +3,38 @@ class Httpie < Formula
 
   desc "User-friendly cURL replacement (command-line HTTP client)"
   homepage "https://httpie.org/"
-  url "https://files.pythonhosted.org/packages/09/8d/581ef7bd9a09dc30b621638a4fa805a2073bbfb45fa06ed37f998f172419/httpie-1.0.2.tar.gz"
-  sha256 "fc676c85febdf3d80abc1ef6fa71ec3764d8b838806a7ae4e55e5e5aa014a2ab"
+  url "https://files.pythonhosted.org/packages/d5/a4/ab61c1dbfdef33c7b7f5f7df0d79eb5cd55a106601a4acc17f983f320b4a/httpie-1.0.3.tar.gz"
+  sha256 "6d1b6e21da7d3ec030ae95536d4032c1129bdaf9de4adc72c596b87e5f646e80"
   head "https://github.com/jakubroztocil/httpie.git"
 
   bottle do
     cellar :any_skip_relocation
     rebuild 1
-    sha256 "4b93be77eb04fbdd3af064849b182520383676b7b03728d94f6867f250d2e35e" => :mojave
```

Comment:
Do not modify the bottle block

Reply:
So, I saw that in the contribution guidelines, though it’s countered by the fact that the script in the httpie project does modify these things (and remove the `rebuild 1` portion, too).

You can see the results in this commit I made over on that project: jakubroztocil/httpie#801

Should I simply ignore what the `make brew-deps` command on that project generated?

Reply:
If it modified the bottle block, yes

Reply:
> the script in the httpie project does modify these things

Modifying the resources blocks is right. Modifying the `bottle do` block is wrong.

Reply:
Would you expand on that, @fxcoudert? I’m not entirely familiar with how bottles are generated, frankly.

Reply:
Would you expand on that, @fxcoudert? I’m not entirely familiar with how bottles are generated, frankly.

Reply:
The bottles are generated by our CI, and the bottle block is modified automatically when we merge a PR. Pull requests should always leave the bottle block untouched.

Reply:
This makes way more sense! I’ll fix this up, stand by…